src/index.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
-const rootDOM = document.getElementById('root');
+const rootDOM = document.getElementById("root");
 const appRoot = ReactDOM.createRoot(rootDOM);
-
 appRoot.render(
-    <h1>My refresher app</h1>,
-    <h2>My First React App</h2>
-);
\ No newline at end of file
+  function Hello(props){
+    return<h1>Hello React First App</h1>
+  }
+);
+
+appRoot.render(<Hello />);
```

Comment:
Excellent!

You just created a _React Component_ and named it Hello! Good job there 👍

Try to pass a prop to your component using an attribute on the JSX tag and receiving this using the `props` parameter. I would like to see what you will make 😁